Assets/PatchKit/UnityEditorExtension/Editor/UI/Window.cs

```diff
@@ -138,7 +138,7 @@ private void OnGUI()
                 currentScreen.Draw();
             }
             catch (ArgumentException)
-            {
+            { 

```

Comment:
Trailing whitespace - you can configure whatever editor you're using to remove those automatically.

Reply:
this code will be changed by Tomas